homework3/hw3/Helpers/LeagueSection.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  LeagueSection.swift
+//  hw3
+//
+//  Created by Tina Jureško on 20.03.2025..
+//
+
+import Foundation
+import UIKit
+import SofaAcademic
+import SnapKit
```

Comment:
Ovdje ti ne trebaju importi vezani za UI. 

Probaj uvijek razmisliti što ti stvarno treba u fielu za import. Compiler nas to ne forsa u Xcode, ali dobra je praksa.